boards/arm/adi_eval_adin1110ebz/adi_eval_adin1110ebz_defconfig

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_BOARD_ADI_EVAL_ADIN1110EBZ=y
+CONFIG_SOC_SERIES_STM32L4X=y
+CONFIG_SOC_STM32L4S5XX=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# Enable Clocks
+CONFIG_CLOCK_CONTROL=y
+
+# enable pin controller
+CONFIG_PINCTRL=y
+
+# enable I2C
+CONFIG_I2C=y
+
+# enable SPI
+CONFIG_SPI=y
+
+# enable flash
+CONFIG_FLASH=y
+
+# enable sensor
+CONFIG_SENSOR=y
+
+# enable FMC
+CONFIG_MEMC=y
+CONFIG_MEMC_STM32=y
+CONFIG_MEMC_STM32_NOR_PSRAM=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_ETH_DRIVER=y
+
+# ADIN1110, driver defaults to Generic SPI, CRC enabled
+# Set CONFIG_ETH_ADIN2111_SPI_CFG0=n to disable CRC
+CONFIG_MDIO=y
+CONFIG_MDIO_SHELL=n
+CONFIG_MDIO_ADIN2111=y
+CONFIG_ETH_ADIN2111=y
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_STATISTICS_ETHERNET=y
```

Comment:
This enables these features for all applications by default, including hello_world. Please remove.

See https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#general-recommendations,
"Unless explicitly recommended otherwise by this section, leave peripherals and their drivers disabled by default."

Reply:
Sorry, the board is designed to test and develop on the ADIN1110, this is why i left the ethernet part enabled. 
Also, i tested i2c temperature sensor and psram to work properly, so, since they are chip mounted onboard that have to work, i thought it was the case to set them in the defconfig. Btw, not a problem, please confirm and i can remove them all.

Reply:
It would be better to add these peripherals in supported in .yml this way you can run twister tests and peripherals can be tested this way